bundle/edu.gemini.itc/src/main/java/edu/gemini/itc/gmos/GmosRecipe.java

```diff
@@ -403,7 +403,7 @@ private static SpcChartData createGmosChart(final SpectroscopyResult[] results,
             }
         }
 
-        return new SpcChartData(type, title, "Wavelength (nm)", yAxis, JavaConversions.asScalaBuffer(data));
+        return new SpcChartData(type, title, "Wavelength (nm)", yAxis, JavaConversions.asScalaBuffer(data).toList());
```

Comment:
For some reason I never thought to use `JavaConversions` from Java code but that seems like a good idea.  `JavaConversions` for Java code that needs to work with Scala collections and `JavaConverters` for Scala code that needs Java collections.